### DISCUSSION TITLE
Import of Open API specification

### DISCUSSION THREAD

**Discussion by sageTimNewton** [mockapi-io/docs#3] [Ideas]
Hi,

Just trying out mockapi and really like it. It does seem to lack one feature that would be nice though. It would be great if you could create a project by importing an Open API spec (https://swagger.io/specification/). This is how another mocking framework (prism) works. This has the following benefits :-
1) No manual configuration of endpoints needed by standardising and automate the endpoint configuration.
2) Allow existing documentation (open api specs) to be used quickly with mockapi.

Let me know your thoughts on this, thanks

Tim

**Comment by nsmirn0v** [comment]
Great suggestion! I will need to think about it :)

**Comment by Sigularusrex** [comment]
Coming here to support @sageTimNewton 's suggestion. I would help a lot to get set up

**Reply by kosytedy** [reply]
www.mockapi.com supports it, including Postman collections

**Comment by kosytedy** [comment]
www.mockapi.com supports it, including Postman collections
